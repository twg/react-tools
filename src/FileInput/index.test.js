/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'
import cases from 'jest-in-case'
import { Button } from './'

cases(
  'matches snapshots',
  opts => {
    expect(
      renderer.create(<Button {...opts.props}>Label</Button>).toJSON()
    ).toMatchSnapshot()
  },
  [
    { name: 'primary', props: { size: 'primary', color: 'orange' } },
    { name: 'hero', props: { size: 'hero', color: 'orange' } },
    { name: 'secondary', props: { size: 'secondary', color: 'orange' } },
    {
      name: 'noMargin',
      props: { size: 'primary', color: 'orange', noMargin: true }
    }
  ]
)
