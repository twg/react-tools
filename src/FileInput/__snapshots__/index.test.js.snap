// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots hero 1`] = `
<button
  className="button hero"
  onClick={undefined}
  style={
    Object {
      "background": "orange",
    }
  }
  type="button"
>
  Label
</button>
`;

exports[`matches snapshots noMargin 1`] = `
<button
  className="button primary noMargin"
  onClick={undefined}
  style={
    Object {
      "background": "orange",
    }
  }
  type="button"
>
  Label
</button>
`;

exports[`matches snapshots primary 1`] = `
<button
  className="button primary"
  onClick={undefined}
  style={
    Object {
      "background": "orange",
    }
  }
  type="button"
>
  Label
</button>
`;

exports[`matches snapshots secondary 1`] = `
<button
  className="button secondary"
  onClick={undefined}
  style={
    Object {
      "background": "orange",
    }
  }
  type="button"
>
  Label
</button>
`;
